liferay-workspace/apps/service-builder/jdbc/jdbc-service/build.gradle

```diff
@@ -8,7 +8,7 @@ dependencies {
 
 	testCompile group: "biz.aQute.bnd", name: "biz.aQute.remote.api", version: "3.3.0"
 	testCompile group: "com.google.guava", name: "guava", version: "23.0"
-	testCompile group: "com.liferay.arquillian", name: "com.liferay.arquillian.arquillian-container-liferay", version: "1.0.6"
+	testCompile group: "com.liferay", name: "com.liferay.arquillian.extension.junit.bridge", version: "1.0.6"
```

Comment:
I don't think we can make these changes, even though I know it is breaking SF.  @lawrence-lee   how did these pass SF in the past?  Perhaps we need to add a exclusion?

Reply:
they mustve made SF even more strict. It used to not care about the build.gradle